sql-cli/sql_cli/configuration.py

```diff
@@ -55,16 +41,16 @@ def get_global_config_filepath(self) -> Path:
         """
         return self.project_dir / CONFIG_DIR / GLOBAL_CONFIG / CONFIG_FILENAME
 
-    @staticmethod
-    def from_yaml_to_dict(filepath: Path) -> dict[str, Any]:
+    def from_yaml_to_dict(self, filepath: Path) -> dict[str, Any]:
         """
         Return a dict with the contents of the given configuration.yaml
 
         :param filepath: Path of the desired configuration.yaml to read contents from.
 
         :returns: Content of the YAML configuration file as a python dictionary.
         """
-        with open(filepath) as fp:
+        load_dotenv(self.project_dir / ".env")
```

Comment:
load environment variables from the initialised project's .env file to expand the YAML with env vars's values in the subsequent lines.